libs/cart/src/drivers/interfaces/cart-service.interface.ts

```diff
@@ -1,9 +1,14 @@
+import { InjectionToken } from '@angular/core';
 import { Observable } from 'rxjs';
-
 import { DaffCart } from '../../models/cart';
+import { DaffCartServiceInterface } from '../interfaces/cart-service.interface';
 
 export interface DaffCartServiceInterface<T extends DaffCart> {
-  get(): Observable<T>;
-  addToCart(productId: string, qty: number): Observable<T>;
-  clear(): Observable<void>;
+	get(): Observable<T>;
```

Comment:
Doesn't `get()` need a cart id for an input?

Reply:
There should only ever be one cart in the client so we could query the cart id from a service `this.cartService.cartId.pipe(...`. However passing in the cart id would make the driver implementation simpler.

Reply:
I don't think the driver should access redux state. The effects layer should call the driver with state if need be.

Reply:
I agree to that. It makes sense for the driver to be a clear "in-and-out" paradigm without having to reach into various aspects of Daffodil.

Reply:
@lderrickable I added the `cart_id`.